engine_redis_test.go

```diff
@@ -605,34 +608,59 @@ func TestRedisEngineHandlePubSubMessage(t *testing.T) {
 	require.True(t, leaveHandlerCalled)
 }
 
+func BenchmarkRedisExtractPushData(b *testing.B) {
+	data := []byte(`__p1:16901:xght__\x12\nchat:index\x1aU\"\x0e{\"input\":\"__\"}*C\n\x0242\x12$37cb00a9-bcfa-4284-a1ae-607c7da3a8f4\x1a\x15{\"name\": \"Alexander\"}\"\x00`)
```

Comment:
do we need `\"` here? looks like the string is already in backquotes

Reply:
This does not matter for this specific bench - since it only deals with `__p1:16901:xght__` part, I just copy-pasted this payload from Redis monitor without any changes.